// danielbonifacio-sdk/dist/errors/index.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
exports.IncomprehensibleMessageError = exports.GenericError = exports.BusinessError = exports.ResourceNotFoundError = exports.SystemError = exports.InvalidDataError = exports.ForbiddenError = exports.ERRORS = void 0;
exports.ERRORS = {
    FORBIDDEN: "https://api.alganews.com.br/forbidden",
    INVALID_DATA: "https://api.alganews.com.br/invalid-data",
    SYSTEM_ERROR: "https://api.alganews.com.br/system-error",
    INVALID_PARAMETER: "https://api.alganews.com.br/invalid-parameter",
    INCOMPREHENSIBLE_MESSAGE: "https://api.alganews.com.br/incomprehensible-message",
    RESOURCE_NOT_FOUND: "https://api.alganews.com.br/resource-not-found",
    BUSINESS_ERROR: "https://api.alganews.com.br/business-error",
};
var Forbidden_error_1 = require("./Forbidden.error");
Object.defineProperty(exports, "ForbiddenError", { enumerable: true, get: function () { return __importDefault(Forbidden_error_1).default; } });
var InvalidData_error_1 = require("./InvalidData.error");
Object.defineProperty(exports, "InvalidDataError", { enumerable: true, get: function () { return __importDefault(InvalidData_error_1).default; } });
var System_error_1 = require("./System.error");
Object.defineProperty(exports, "SystemError", { enumerable: true, get: function () { return __importDefault(System_error_1).default; } });
var ResourceNotFound_error_1 = require("./ResourceNotFound.error");
Object.defineProperty(exports, "ResourceNotFoundError", { enumerable: true, get: function () { return __importDefault(ResourceNotFound_error_1).default; } });
var Business_error_1 = require("./Business.error");
Object.defineProperty(exports, "BusinessError", { enumerable: true, get: function () { return __importDefault(Business_error_1).default; } });
var Generic_error_1 = require("./Generic.error");
Object.defineProperty(exports, "GenericError", { enumerable: true, get: function () { return __importDefault(Generic_error_1).default; } });
var IncomprehensibleMessage_error_1 = require("./IncomprehensibleMessage.error");
Object.defineProperty(exports, "IncomprehensibleMessageError", { enumerable: true, get: function () { return __importDefault(IncomprehensibleMessage_error_1).default; } });
